### DISCUSSION TITLE
Rustdesk network deploying

### DISCUSSION THREAD

**Discussion by Josep-San** [rustdesk/rustdesk#5636] [Q&A]
Hi there,
I'm trying to deploy rustdesk using a powershell script on Windows. Following the documentation on rustdesk.com (https://rustdesk.com/docs/en/self-host/client-deployment/) and adapting it to my needs I ended with a script that successfully installs rustdesk service but I'm not able to change the network configuration.
When I try to do this:
```
net stop rustdesk
# Where $rustdesk_cfg variable contains the configuration string.
.\rustdesk.exe --config $rustdesk_cfg
net start rustdesk

```
From the Powershell script or a command line shell with administration rights there's no effect. In the documentation the string configuration is described as Base64 and inversed. I've tried Base64 inversed and not inversed and there no change. Nothing happens on the network configuration.
If I copy the string to the clipboard and import it to the rustdesk GUI network tab it works, so I'm confused and don't know where to look on forward.
Any idea? Am I doing something wrong? It is not intended to work like this?

**Comment by dinger1986** [comment]
are you using pro?

**Reply by Josep-San** [reply]
Nope. I've tried with 1.2.0 and 1.2.2 just in case, but I don't have a license for the Pro version.

**Reply by dinger1986** [reply]
doesnt work with nightly build either, this was working prior to 1.2.2 being released

**Comment by dinger1986** [comment]
filled bug report https://github.com/rustdesk/rustdesk/issues/5637

**Comment by dinger1986** [comment]
for now use this https://rustdesk.com/docs/en/self-host/client-deployment/integrations/tactical-rmm/#install-script-replace-ipaddress-and-key

**Reply by Josep-San** [reply]
Indeed.
Thanks @dinger1986 
I just tried that other method and it worked. For me that's enough.

**Reply by dinger1986** [reply]
ok great glad you are working ok

**Comment by dinger1986** [comment]
working fine, needed to remove the = from the string and works fine, also service needs to be running

**Comment by Josep-San** [comment]
That's something I didn't try ... Documentation needs clarification on that matter.
Thanks a lot for the support.
I don't know if I want to go back to the --config version or stay with the Tactical RMM method. Any suggestions?

**Reply by dinger1986** [reply]
docs have been updated https://rustdesk.com/docs/en/self-host/client-configuration/#automatic-config since, I like the --config option as its a one liner but up to you, both work well
